src/Traits/ResolveBSN.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace OWC\Zaaksysteem\Traits;
+
+use function OWC\Zaaksysteem\Foundation\Helpers\decrypt;
+use function Yard\DigiD\Foundation\Helpers\resolve;
+
+trait ResolveBSN
```

Comment:
Ik vind een `trait` niet de juiste vorm voor deze functionaliteit. Het maakt het geheel lastig testbaar, omdat je nu niet een aparte/gemockte `BSNResolver` class (o.i.d.) kunt injecteren. 

 Ik zou dit in een losse class zetten en ophalen via de `Container`. Je kan dan (bijvoorbeeld) ook een `TestBSNResolver` class toevoegen die een vast BSN nummer teruggeeft om het e.e.a. lokaal te testen.

Reply:
Is toegevoegd aan de container middels de DigiDSegmentResolver klasse.